SPECS/curl/CVE-2023-38545.patch

```diff
@@ -1,111 +0,0 @@
-From 061532e794feabd3fe9b2f2e1b802dd471d0da65 Mon Sep 17 00:00:00 2001
```

Comment:
Fix present in 8.5.0: https://github.com/curl/curl/commit/fb4415d8aee6c1045be932a34fe6107c2f5ed147